.github/actions/trivy/action.yml

```diff
@@ -4,16 +4,110 @@ description: Trivy docker container vulnerability assessment
 inputs:
   IMAGE:
     required: true
-  TRIVY_IGNORES:
-    required: false
-    
+
 runs:
   using: "composite"
   steps:
-  - uses: lazy-actions/gitrivy@v3
+
+  - uses: aquasecurity/trivy-action@master
+    id: scanCritical
+    continue-on-error: true
+    with:
+      image-ref: ${{ inputs.IMAGE }}
+      template: "@actions/.github/actions/trivy/template.tpl"
+      format: template
+      severity: 'CRITICAL'
+      exit-code: '1'
+      ignore-unfixed: true
+      vuln-type: 'os,library'
+      security-checks: vuln
+      output: container_scan.txt
+
+  - name: Load plan file
+    id: readCritical
+    if: steps.scanCritical.outcome != 'success'
+    uses: juliangruber/read-file-action@v1
+    with:
+      path: container_scan.txt 
+
+  - name: Create Issue on container scan
+    uses: dacbd/create-issue-action@v1
+    id: initialIssue
+    if: steps.scanCritical.outcome != 'success'
+    with:
+      title: CRITICAL Vulnerability Detected ${{inputs.IMAGE_TAG }}
+      token: ${{ env.PAT }}
+      repo: ci-testing
+      body: >
+        ### Context
+        [Failed Run](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
+        [Codebase](https://github.com/${{ github.repository }}/tree/${{ github.sha }})
+        Workflow name - `${{ github.workflow }}`
+        Job -           `${{ github.job }}`
+        status -        `${{ job.status }}`
+        Vulnerabilities -
+        
+        ${{ steps.readCritical.outputs.content }}
+      assignees: ${{ github.actor }}
+      labels: team/devops,security,vulnerability,docker
```

Comment:
We should probably use the existing `area/security` label in place of `security`. We could also qualify `vulnerability` as `type/vulnerability` but I'll leave that up to your discretion.